.github/workflows/pull_requests.yml

```diff
@@ -30,3 +30,9 @@ jobs:
         with:
           name: jdk17-test-results
           path: "**/target/*-reports*/**/TEST-*.xml"
+  e2e:
+    needs: jdk17
+    uses: ryanemerson/gingersnap-test/.github/workflows/e2e.yaml@test_dev
```

Comment:
@pminz This will be updated to https://github.com/gingersnap-project/e2e once it's ready.